context/app/static/js/pages/Services/Services.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Paper from '@material-ui/core/Paper';
+
+import SectionHeader from 'js/components/Detail/SectionHeader';
+import SectionContainer from 'js/components/Detail/SectionContainer';
```

Comment:
Yes, I think they can be moved into `shared-styles`.

Reply:
Done. Touched a lot of other files in the process, but I think the end result makes sense.